.travis.yml

```diff
@@ -0,0 +1,44 @@
+dist: bionic
+language: python
+python:
+  - "3.8"
+
+stages:
+  - lint
+  - test
+
+jobs:
+  include:
+    - stage: lint
+      env:
+        - HADOLINT: "${HOME}/hadolint"
+      install:
+        - gem install mdl
+        - curl -sL -o ${HADOLINT} "https://github.com/hadolint/hadolint/releases/download/v1.17.6/hadolint-$(uname -s)-$(uname -m)"
+        - chmod 700 ${HADOLINT}
+      script:
+        - mdl README.md
```

Comment:
`make chech` seens to be missing

Reply:
It is in a different job down below. This job only validates README.md and Dockerfile.

Reply:
Seems weird: `make check` does linting, not testing.

But ok.